frontend/src/variables.css

```
@@ -1,8 +1,19 @@
 @import "open-props/style";
+@import "open-props/colors-hsl";
+
 
 :root {
   --white: #fff;
   --black: #000;
+  
+  --color-primary: var(--yellow-5);
+  --color-primary-light: hsl(var(--yellow-5-hsl) / 30%);
```

Du coup utilise la variable color primary

```suggestion
  --color-primary-light: hsl(var(--color-primary) / 30%);
```